android/assets/jsons/translations/Spanish.properties

```diff
@@ -5530,174 +5529,175 @@ Often this results in the city immediately converting to their religion = A menu
-when below [amount] Happiness = 
+when below [amount] Happiness = cuando por debajo de[amount] felicidad
  # Requires translation!
-during the [era] = 
+during the [era] = durante el [era]
  # Requires translation!
-before the [era] = 
+before the [era] = antes de [era]
  # Requires translation!
-starting from the [era] = 
+starting from the [era] = a partir de la [era]
  # Requires translation!
-after discovering [tech] = 
+after discovering [tech] = después de descubrir [tech]
  # Requires translation!
-before discovering [tech] = 
+before discovering [tech] = antes de descubrir [tech]
  # Requires translation!
-after adopting [policy] = 
+after adopting [policy] = después de adoptar [policy]
  # Requires translation!
-before adopting [policy] = 
+before adopting [policy] = antes de adoptar [policy]
 if this city has at least [amount] specialists = si esta ciudad tiene por lo menos [amount] especialistas
  # Requires translation!
-in cities where this religion has at least [amount] followers = 
+in cities where this religion has at least [amount] followers = en ciudades donde esta religión tiene al menos [amount] seguidores
  # Requires translation!
-with a garrison = 
+with a garrison = con una guarnición
  # Requires translation!
-for [mapUnitFilter] units = 
+for [mapUnitFilter] units = por [mapUnitFilter] unidades
  # Requires translation!
-vs cities = 
+vs cities = vs ciudades
  # Requires translation!
-vs [mapUnitFilter] units = 
+vs [mapUnitFilter] units = vs [mapUnitFilter] unidades
  # Requires translation!
-when fighting units from a Civilization with more Cities than you = 
+when fighting units from a Civilization with more Cities than you = cuando luches contra unidades de una Civilización con más Ciudades que tú
  # Requires translation!
-when attacking = 
+when attacking = al atacar
  # Requires translation!
-when defending = 
+when defending = al defender
  # Requires translation!
-when fighting in [tileFilter] tiles = 
+when fighting in [tileFilter] tiles = al pelear en [tileFilter] mosaicos
  # Requires translation!
-on foreign continents = 
+on foreign continents = en continentes extranjeros
  # Requires translation!
-when adjacent to a [mapUnitFilter] unit = 
+when adjacent to a [mapUnitFilter] unit = cuando está adyacente a una [mapUnitFilter] unidad
  # Requires translation!
-when above [amount] HP = 
+when above [amount] HP = cuando está por encima de [amount] HP
  # Requires translation!
-when below [amount] HP = 
+when below [amount] HP = cuando está por debajo de [amount] HP
  # Requires translation!
-with [amount] to [amount2] neighboring [tileFilter] tiles = 
+with [amount] to [amount2] neighboring [tileFilter] tiles = con [amount] a [amount2] [tileFilter] fichas
  # Requires translation!
-with [amount] to [amount2] neighboring [tileFilter] [tileFilter2] tiles = 
+with [amount] to [amount2] neighboring [tileFilter] [tileFilter2] tiles = con [amount] a [amount2] [tileFilter] [tileFilter] mosaicos
```

Comment:
Here. The second tileFilter needs a '2' on the end :)

Reply:
thanks

Reply:
how can i edit it again?

Reply:
I already learned to edit

Reply:
I already resolved it

Reply:
thanks for checking the document